content/en/docs/tasks/administer-cluster/coredns.md

```diff
@@ -16,16 +16,29 @@ This page describes the CoreDNS upgrade process and how to install kube-dns inst
 
 {{% capture steps %}}
 
-## Installing kube-dns with kubeadm
+## About CoreDNS
 
-In Kubernetes 1.11, [CoreDNS](https://coredns.io) has graduated to General Availability (GA)
+[CoreDNS](https://coredns.io) is a flexible, extensible authoritative DNS server that can run on a Kubernetes cluster.
+The CoreDNS project is hosted, like Kubernetes, by the [CNCF.](http://www.cncf.io)
```

Comment:
Two options here; pick one:

Remove ", like Kubernetes," altogether (my suggestion, because context doesn't require repeating information users are highly likely to have already here, and because it's irrelevant to using CoreDNS)

or, if you really need to keep it: "Like Kubernetes, the CoreDNS project is hosted by the [CNCF](http://www.cncf.io)." (still a complex sentence, but easier to scan bc removes parenthetical phrase)